README.md

```diff
@@ -219,7 +219,7 @@ This package updates itself using a daily cronjob running on GitHub Actions. The
 generated by [`bin/octokit-webhooks.js`](bin/octokit-webhooks.js). Run
 `node bin/octokit-webhooks --usage` for instructions.
 
-The update script is scraping [GitHub’s Webhooks Event Types & Payloads](https://developer.github.com/v3/activity/events/types/) documentation page and extracts the meta information using [cheerio](https://www.npmjs.com/package/cheerio).
+The update script is scraping [GitHub’s Webhooks Event Types & Payloads](https://docs.github.com/v3/activity/events/types/) documentation page and extracts the meta information using [cheerio](https://www.npmjs.com/package/cheerio).
```

Comment:
https://docs.github.com/v3/activity/events/types/ does redirect, but I think we should use the current docs URL for event types

```suggestion
The update script is scraping [GitHub’s Webhooks Event Types & Payloads](https://docs.github.com/en/free-pro-team@latest/developers/webhooks-and-events/webhook-events-and-payloads) documentation page and extracts the meta information using [cheerio](https://www.npmjs.com/package/cheerio).
```